swift/ql/lib/codeql/swift/frameworks/StandardLibrary/Data.qll

```diff
@@ -0,0 +1,6 @@
+import swift
+private import codeql.swift.dataflow.ExternalFlow
+
+private class DataSummaries extends SummaryModelCsv {
+  override predicate row(string row) { row = ";Data;true;init(_:);;;Argument[0];ReturnValue;taint" }
```

Comment:
> These classes need proper modeling, which deserve their own PRs.

I agree - are you planning to do this?  Shall I create an issue for it?

Reply:
I can work on that after finishing this query if you think it has enough priority. An issue for it would be nice in any case.

Reply:
I do think its important.  Do you think covering `NSData` as well should have similar or lower priority?

Reply:
I've created https://github.com/github/codeql-c-team/issues/1334 for modelling `Data`; please update the description as you see fit.

Reply:
> Do you think covering NSData as well should have similar or lower priority?

I think they probably should have similar priority.

Reply:
I'll create an issue for `NSData` (and `NSString`) as well then.